activation/poet.go

```diff
@@ -126,16 +129,42 @@ func WithLogger(logger *zap.Logger) PoetClientOpts {
 	}
 }
 
+func customLinearJitterBackoff(min, max time.Duration, attemptNum int, resp *http.Response) time.Duration {
+	attemptNum++
+	if max <= min {
+		return min * time.Duration(attemptNum)
+	}
+
+	source := rand.New(rand.NewSource(int64(time.Now().Nanosecond())))
+
+	jitter := source.Float64() * float64(max-min)
+	jitterMin := int64(jitter) + int64(min)
+
+	sleep := time.Duration(jitterMin * int64(attemptNum))
```

Comment:
With `math/rand/v2` this can be written as:

```suggestion
	jitter := min + rand.N(max-min)
	sleep := jitter * time.Duration(attemptNum)
```